src/pages/publication/new_publication/LoadPublication.tsx

```diff
@@ -35,7 +35,7 @@ const LoadPublication: FC<LoadPublicationProps> = ({ expanded, onChange, openFor
   const dispatch = useDispatch();
 
   useEffect(() => {
-    if (uppy && !uppy.hasUploadSuccessEventListener) {
+    if (uppy) {
```

Comment:
mistenker mulig ny bug her...
Hvis man laster opp 1 fil, så en fil til etterpå, vil det ikke komme 2stk av den siste fila da, slik at det listes opp 3 filer?

Reply:
<img width="1680" alt="Skjermbilde 2020-05-11 kl  15 25 43" src="https://user-images.githubusercontent.com/6595982/81566875-c7d41380-939b-11ea-9b31-4ff5712e99cc.png">

Jo, det er ikke helt bra det der. Filene listes ikke dobbelt, men i UppyDashboard så ser det ut som den laster opp alle filene på nytt. Må nok fikses ja. Tar den tilbake til selected 😢

Reply:
lol. nevermind. Det er sånn Uppy fungerer. Det står `upload 1+ files` men det betyr `upload one more file` - så de blir ikke lagt til dobbelt i uppy state allikevel (som jeg trodde). Så da er det ikke noen bug med dette allikevel.

Reply:
det var litt overraskende faktisk... eneste grunnen til å sjekke `hasUploadSuccessEventListener` var jo å unngå at man la til en ny `upload-success`-event for hver gang, slik at fil nummer _n_ fikk _n_ upload-success events.  men den blir sikkert ryddet opp i cleanupen der uansett da

Reply:
ja, det virker sånn. Vet ikke helt hva som skjer "under the hood" jeg, men det fungerer hvertfall nå (bortsett fra warnings i console som vi må fikse etterhvert da)